oneview-ansible.md

```diff
@@ -2563,15 +2563,15 @@ Manage OneView Fibre Channel Network resources.
 | ------------- |-------------| ---------|----------- |--------- |
 | config  |   No  |  | |  Path to a .json configuration file containing the OneView client configuration. The configuration file is optional. If the file path is not provided, the configuration will be loaded from environment variables.  |
 | data  |   Yes  |  | |  List with the Fibre Channel Network properties.  |
-| state  |   |  | <ul> <li>present</li>  <li>absent</li> </ul> |  Indicates the desired state for the Fibre Channel Network resource. 'present' will ensure data properties are compliant with OneView. 'absent' will remove the resource from OneView, if it exists.  |
+| state  |   |  | <ul> <li>present</li>  <li>absent</li> </ul> |  Indicates the desired state for the Fibre Channel Network resource. **present** will ensure data properties are compliant with OneView. **absent** will remove the resource from OneView, if it exists.  |
```

Comment:
Open question. Isn't better the inline code markdown instead?

**Option 1 - Inline code**

Indicates the desired state for the Fibre Channel Network resource. `present` will ensure data properties are compliant with OneView. `absent` will remove the resource from OneView, if it exists.

**Option 2 - Emphasis**

Indicates the desired state for the Fibre Channel Network resource. **present** will ensure data properties are compliant with OneView. **absent** will remove the resource from OneView, if it exists.